sandy/core/cov.py

```diff
@@ -474,57 +474,80 @@ def get_corr(self):
             )
         return self.__class__(df)
 
-    def invert(self, rows=None):
+    def invert(self):
         """
         Method for calculating the inverse matrix.
 
-        Parameters
-        ----------
-        tables : `bool`, optional
-            Option to use row calculation for matrix calculations. The
-            default is False.
-        rows : `int`, optional
-            Option to use row calculation for matrix calculations. This option
-            defines the number of lines to be taken into account in each loop.
-            The default is None.
-
         Returns
         -------
-        `CategoryCov`
+        `pandas.DataFrame`
             The inverse matrix.
 
-        Examples
-        --------
-        >>> S = sandy.CategoryCov(np.diag(np.array([1, 2, 3])))
-        >>> S.invert()
+        Notes
+        -----
+        Many covariance matrices for nuclear data are ill-defined and might
+        have condition numbers that make the matrix inversion process
+        impossible.
+        To make up for this limitation we produce the (Moore-Penrose)
+        pseudo-inverse of a Hermitian matrix, as implemented in `numpy`.
+        Default options are used.
+        This method does not require any pre-processing of the covariance
+        data, e.g. removing zeros from the matrix diagonal or truncating
+        eigenvalues.

```

Comment:
In any case we don't need to truncate the eigenvalues to invert a matrix. `np.linalg.pinv()` allows us to invert singular (even not squared) matrix, while 'np.linalg.inv()' does not